hydroshare/local_settings.py

```diff
@@ -169,6 +169,11 @@
 HS_WWW_IRODS_ZONE = ''
 HS_USER_IRODS_ZONE = 'hydroshareuserZone'
 
+# Info needed for mygeohub MultiSpec tool launch from HydroShare
```

Comment:
I don't see a reason why these couldn't be placed in settings with the real values.  This isn't sensitive information.

Reply:
@sblack-usu The reason I put those info in local_settings.py is that all these info are mygeohub iRODS production server settings, such as iRODS path that includes iRODS production zone name, and iRODS resource name, so they are indeed sensitive info in the same way that we don't expose iRODS production server configurations such as iRODS zone and resource name for HydroShare iRODS production server. The GEOHUB_HOMEPAGE_URL can be moved into settings.py, though. So I'll move that URL setting into settings.py but leave two iRODS production server configuration values in local_settings.py.